activemodel/lib/active_model/secure_password.rb

```diff
@@ -29,6 +46,13 @@ module ClassMethods
       # it). When this attribute has a +nil+ value, the validation will not be
       # triggered.
       #
+      # Passwords get easier to crack as CPUs get faster and faster. The
+      # default cost factor is bumped every 18 months to scale with attacker
+      # CPU power. When a password is authenticated and found to have an old
+      # cost factor, it's automatically updated to the current cost,
+      # ensuring that passwords set years ago stay hard to crack so long
+      # as they are regularly authenticated.
+      #
```

Comment:
Need to consider ramifications of writing to db in a context where only reads are expected. App may be authenticating using a read-only conn to a db replica.